src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -24,12 +18,16 @@
 using Windows.UI.Core;
 using SortDirection = Files.Shared.Enums.SortDirection;
 
-namespace Files.App.Views
+namespace Files.App.Views.Shells
```

Comment:
We're moving away from the concept of "shells".

Reply:
this pr's aiming to avoid use root namespace.